### ISSUE TITLE
Bibdocfile: OCR text-file is not deleted when related version of full-text file is deleted.

### ISSUE THREAD

**Issue by Kennethhole** [inveniosoftware/invenio#3519]
The PURGE function in bibupload deletes the old versions of a file, but the .text;1 file which contains the OCR for the previous version is not deleted.

The .text;1 file is dropped from the indexer with once the .text;2 file is created and is just stored as an unused file. As the file has no function it should be deleted.

```
-rw-r--r--  1 apache apache 122079 sep  9 10:59 content.jpg;icon;2
-rw-r--r--  1 apache apache 179315 sep  9 10:59 content.pdf;2
-rw-r--r--  1 apache apache    202 sep  9 10:59 .md5
-rw-r--r--  1 apache apache     19 sep  3 19:55 .reclinks
-rw-r--r--  1 apache apache   8871 sep  3 19:56 .text;1
-rw-r--r--  1 apache apache   8892 sep  9 10:59 .text;2
```

**Comment by kaplun** [comment]
Actually the whole bibdoc directory could be happily removed recursively when the corresponding BibDoc is expunged.

**Comment by tiborsimko** [comment]
@Kennethhole @kaplun Any updates on this? Solved locally within TIND? I'm moving the issue to v1.1.8.

**Comment by kaplun** [comment]
@tiborsimko was faster to implement the fix, than to reply I had no news about it :dancer:

**Comment by tiborsimko** [comment]
Closed by #3719.
